docs/documentation/integrations.md

```diff
@@ -15,8 +15,10 @@ Usually, this involves a few steps:
   See [Markdown macros](#markdown-macros) for further details.
 
 * Make sure you've added your integration to
-  `zerver/lib/integrations.py`; this results in your integration
-  appearing on the `/integrations` page.
+  `zerver/lib/integrations.py` in both the `WEBHOOK_INTEGRATIONS` section and
+  the `DOC_SCREENSHOT_CONFIG` section; this results in your integration
+  appearing on the `/integrations` page  and enables the screenshot to be
+  automatically generated.
```

Comment:
Very minor nitpick but would be simpler to just say "; this results in your integration appearing on the `/integrations` page and generates a sample screenshot automatically."

Reply:
@eeshangarg I chose that wording because the screenshot will only be generated automatically if they run the generate screenshot tool, so the wording suggest wouldn't technically be accurate.

Reply:
Yeah, I think I agree with the original wording for that reason.  (Though I did split this into two sentences for general readability after adding mention of `INTEGRATIONS` for non-webhook integrations).